skimage/_shared/fft.py

```diff
@@ -0,0 +1,20 @@
+"""Prefer FFTs via the new scipy.fft module when available (SciPy 1.4+)
```

Comment:
Please, could you add an entry in TODO.txt to update this code when our min req will be 1.4+?

Thank you!

Reply:
In that file, you can also remove one entry
```
"* When minimum supported version of ``scipy`` reaches 0.18, use
  ``scipy.fftpack.next_fast_len`` directly in
  ``skimage.feature.masked_register_translation``."
```

Reply:
I restored the import from scipy.fftpack.next_fast_len for scipy < 0.18 and have just updated the path in that entry (see commit 753204c)